wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/Utils.java

```diff
@@ -37,12 +38,8 @@ public static String getDevMachineName(Environment envConfig) {
         for (Map.Entry<String, ? extends MachineConfig> entry : envConfig.getMachines().entrySet()) {
             // TODO should we use server ref instead of installers?
             List<String> installers = entry.getValue().getInstallers();
-            if (installers != null) {
-                for (String installerKey : installers) {
-                    if (installerKey.equals(WSAGENT_INSTALLER) || installerKey.startsWith(WSAGENT_INSTALLER + ":")) {
```

Comment:
because I already deleted it in my branch and there is no sense to double this piece of work in Anatoliy's PR. I will create a PR shortly